tools/src/dbgen/CLI.hs

```diff
@@ -34,7 +31,11 @@ data CLI = CLI
     -- ^ If true, print the stats for the `wallet-db`
     , queryMethod       :: Maybe Method
     -- ^ If true, generate a DB targeting mainnet.
+    , addTo             :: Maybe AccountId
+    -- ^ If specified, only append addresses to the
+    -- given <wallet_id@account_id>
     , genFakeUtxo       :: Bool
+    -- ^ If true, generate fake UTXO.
```

Comment:
I would prefer we extend the Dhall config file we have, rather than adding another CLI command. What about something like, for example:

```
{ wallet_spec = { account_spec = { addresses = 1 }, accounts = 1, balance = 10000 }
, wallets     = 1
}
```

This would make sure that the total Utxo for this wallet would be 10000, randomly divided into the addresses is being asked to generate.